dgm-builder/src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -294,7 +297,9 @@ private void translateMethod(final CompilationUnitTree cut, ExecutableElement e,
         }
         delegatingParams.forEach(p -> delegateCall.arg(p));
 
-        JVar $b = m.body().decl($Builder, "b", JExpr._new($Builder).arg(JExpr.invoke("loc").arg(methodName)));
+        JVar untrustedB = m.body().decl($Builder, "untrustedB", JExpr._new($Builder).arg(JExpr.invoke("loc").arg(methodName)));
+        JVar $b = m.body().decl($Builder, "b", untrustedB.invoke("contextualize").arg(codeModel.ref("com.cloudbees.groovy.cps.sandbox.Trusted").staticRef("INSTANCE")));
```

Comment:
Wish I could say I understood what this does, but that applies to pretty much everything in this file…anyway yes I think you are on the right track using `Trusted`—I had forgotten it existed. IIRC introduced for the benefit of `workflow-cps-global-lib`.